ui/app/AppLayouts/Chat/ChatColumn/ActivityCenter.qml

```diff
@@ -111,16 +114,33 @@ Popup {
 
                     property int idx: DelegateModel.itemsIndex
 
+                    Component.onCompleted: {
+                        switch (model.notificationType) {
```

Comment:
@emizzle mentioned before that there's no guarantee that models could be in a state where data was already available for use IIRC. Could this happen here?

Reply:
I don't think so, because the List will not be rendered unless there is data